src/storage/AdminProcessor.h

```diff
@@ -20,7 +21,32 @@ class TransLeaderProcessor : public BaseProcessor<cpp2::AdminExecResp> {
     }
 
     void process(const cpp2::TransLeaderReq& req) {
-        UNUSED(req);
+        auto spaceId = req.get_space_id();
+        auto partId = req.get_part_id();
+        auto ret = kvstore_->part(spaceId, partId);
+        if (!ok(ret)) {
+            resp_.set_code(to(error(ret)));
+            promise_.setValue(std::move(resp_));
+            delete this;
+        }
+        auto part = nebula::value(ret);
+        // need to trans from storage addr to raft addr
+        auto host = HostAddr(req.get_new_leader().get_ip(), req.get_new_leader().get_port() + 1);
+        part->asyncTransferLeader(std::move(host),
+                                  [this, spaceId, partId] (kvstore::ResultCode code) {
+            if (code == kvstore::ResultCode::ERR_LEADER_CHANGED) {
```

Comment:
If newLeader is not specified,   we should return OK if current part is not leader